.github/workflows/integration-tests.yml

```
@@ -0,0 +1,72 @@
+
+      - name: Verify Podman installation
+        run: |
+          podman --version
+          podman info
+
+      - name: Build JDT.LS container image with Podman
+        run: |
+          podman build -t jdtls-analyzer:test .
+
+      - name: Run Phase 2 integration tests in container
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2046:warning:2:6: Quote this to prevent word splitting

(shellcheck)